### PR TITLE
API-5390: Migration to hmrc-mongo

### PR DESCRIPTION
**Pull request by syed-ali-hmrc**

### CODE REVIEW

File: it/uk/gov/hmrc/thirdpartydeveloperfrontend/repositories/FlowRepositoryISpec.scala [hunk 1]
```diff
@@ -7,160 +7,158 @@ import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.connectors.{Combine
...
-        fetchLastUpdated(currentFlow).isAfter(lastUpdatedInCurrentFlow) shouldBe true
-        fetchLastUpdated(flowOfDifferentType).isAfter(lastUpdatedInFlowOfDifferentType) shouldBe true
-        fetchLastUpdated(flowInDifferentSession) shouldBe lastUpdatedInFlowInDifferentSession
-      }
-    }
-  }
-}
+import java.time.LocalDateTime
+import scala.concurrent.ExecutionContext.Implicits.global
+//
```
Comment (peteslater-ee) [inline it/uk/gov/hmrc/thirdpartydeveloperfrontend/repositories/FlowRepositoryISpec.scala:18]: Should this all the commented out?

File: app/uk/gov/hmrc/thirdpartydeveloperfrontend/config/SessionTimeoutFilterWithWhitelist.scala [hunk 2]
```diff
@@ -32,8 +32,11 @@ class SessionTimeoutFilterWithWhitelist @Inject()(config: SessionTimeoutFilterCo
   val whitelistedCalls: Set[WhitelistedCall] = Set(WhitelistedCall(loginUrl, "GET"), WhitelistedCall(loginUrl, "POST"))
 
   override def apply(f: (RequestHeader) => Future[Result])(rh: RequestHeader): Future[Result] = {
-    if (whitelistedCalls.contains(WhitelistedCall(rh.path, rh.method))) f(rh)
-    else {
+    if (whitelistedCalls.contains(WhitelistedCall(rh.path, rh.method))) {
+      println("WhiteListed Call")
```
Comment (peteslater-ee) [inline app/uk/gov/hmrc/thirdpartydeveloperfrontend/config/SessionTimeoutFilterWithWhitelist.scala:36]: Remove println

File: app/uk/gov/hmrc/thirdpartydeveloperfrontend/config/SessionTimeoutFilterWithWhitelist.scala [hunk 3]
```diff
@@ -32,8 +32,11 @@ class SessionTimeoutFilterWithWhitelist @Inject()(config: SessionTimeoutFilterCo
...
+      f(rh)
+    } else {
+      println("CALLING SUPER")
```
Comment (peteslater-ee) [inline app/uk/gov/hmrc/thirdpartydeveloperfrontend/config/SessionTimeoutFilterWithWhitelist.scala:39]: Remove println

File: it/uk/gov/hmrc/thirdpartydeveloperfrontend/repositories/FlowRepositoryISpec.scala (hunk 1 above)
Comment (syed-ali-hmrc) [reply it/uk/gov/hmrc/thirdpartydeveloperfrontend/repositories/FlowRepositoryISpec.scala]: Addressed

File: app/uk/gov/hmrc/thirdpartydeveloperfrontend/config/SessionTimeoutFilterWithWhitelist.scala (hunk 2 above)
Comment (syed-ali-hmrc) [reply app/uk/gov/hmrc/thirdpartydeveloperfrontend/config/SessionTimeoutFilterWithWhitelist.scala]: Addressed

File: app/uk/gov/hmrc/thirdpartydeveloperfrontend/config/SessionTimeoutFilterWithWhitelist.scala (hunk 3 above)
Comment (syed-ali-hmrc) [reply app/uk/gov/hmrc/thirdpartydeveloperfrontend/config/SessionTimeoutFilterWithWhitelist.scala]: Addressed

**Review by mattclark-zerogravit — APPROVED** [review]
👍

**Pull request merged**